### DISCUSSION TITLE
Getting updates on releases (GitHub releases)

### DISCUSSION THREAD

**Discussion by hyunbinseo** [sveltejs/kit#3618] [General]
For those who want update notifications on select packages of this mono-repo, reference https://github.com/sveltejs/kit/discussions/3618#discussioncomment-2074623.

---

Finally, new package releases are added to the [GitHub releases](https://github.com/sveltejs/kit/releases) page. 👏 

With this change, it is now possible to receive emails on all `@sveltejs/kit` updates.

That is if `Watch > Custom > Releases` is enabled in the `@sveltejs/kit` [repository](https://github.com/sveltejs/kit).

---

Question is, what packages will be included in the [releases](https://github.com/sveltejs/kit/releases) page.

Currently received emails for `create-svelte` and `kit`.

**Comment by pngwn** [comment]
All packages in the repo will be on the releases page from now. That is `kit`, `create-svelte`, and all adapters (`adapter-*`) as well as any future packages.

**Reply by hyunbinseo** [reply]
Thank you for the information.

I guess I should disable notifications (watch) for releases and stick to [GitHub File Watcher](https://app.github-file-watcher.com/).

**Comment by hyunbinseo** [comment]
For those who want update notifications on select packages of this mono-repo, there seems to be two options.

---

Use RSS feed reader solutions with the following urls.
- [adapter-auto](https://github.com/sveltejs/kit/commits/master/packages/adapter-auto/CHANGELOG.md.atom)
- [adapter-begin](https://github.com/sveltejs/kit/commits/master/packages/adapter-begin/CHANGELOG.md.atom)
- [adapter-cloudflare-workers](https://github.com/sveltejs/kit/commits/master/packages/adapter-cloudflare-workers/CHANGELOG.md.atom)
- [adapter-cloudflare](https://github.com/sveltejs/kit/commits/master/packages/adapter-cloudflare/CHANGELOG.md.atom)
- [adapter-netlify](https://github.com/sveltejs/kit/commits/master/packages/adapter-netlify/CHANGELOG.md.atom)
- [adapter-node](https://github.com/sveltejs/kit/commits/master/packages/adapter-node/CHANGELOG.md.atom)
- [adapter-static](https://github.com/sveltejs/kit/commits/master/packages/adapter-static/CHANGELOG.md.atom)
- [adapter-vercel](https://github.com/sveltejs/kit/commits/master/packages/adapter-vercel/CHANGELOG.md.atom)
- [create-svelte](https://github.com/sveltejs/kit/commits/master/packages/create-svelte/CHANGELOG.md.atom)
- [kit](https://github.com/sveltejs/kit/commits/master/packages/kit/CHANGELOG.md.atom)

---

Use [GitHub File Watcher](https://app.github-file-watcher.com/) with the following configuration.

The repository you want notifications for: `https://github.com/sveltejs/kit` 

The files you want to track (select one):
- `packages/adapter-auto/CHANGELOG.md`
- `packages/adapter-begin/CHANGELOG.md`
- `packages/adapter-cloudflare-workers/CHANGELOG.md`
- `packages/adapter-cloudflare/CHANGELOG.md`
- `packages/adapter-netlify/CHANGELOG.md`
- `packages/adapter-node/CHANGELOG.md`
- `packages/adapter-static/CHANGELOG.md`
- `packages/adapter-vercel/CHANGELOG.md`
- `packages/create-svelte/CHANGELOG.md`
- `packages/kit/CHANGELOG.md`
